bundles/core-combat/player-events.js

```diff
@@ -151,6 +116,61 @@ module.exports = (srcPath) => {
           Broadcast.sayAt(this, '');
           Broadcast.prompt(this);
         }
+
+        function makeAttack(attacker, defender) {
+          const amount = state.RandomUtil.inRange(5, 20);
+          hadActions = true;
+
+          const damage = new Damage({
+            attribute: "health",
+            amount,
+            attacker
+          });
+          damage.commit(defender);
+
+          attacker.combatData.lag = attacker.combatData.speed * 1000;
+        }
+
+        function handleDeath(deadEntity, killer) {
+          deadEntity.combatants.forEach(combatant => {
+            deadEntity.removeCombatant(combatant);
+            combatant.removeCombatant(deadEntity);
+          }, deadEntity);
+
+          if (Reflect.has(deadEntity, 'removePrompt')) {
```

Comment:
I'd prefer this check to be `instanceof Player` or alternatively `TypeUtil.is(deadEntity, Promptable)` if you're only looking for the addPrompt/removePrompt